libgalois/src/analytics/louvain_clustering/dist_friendly_louvain_clustering.cpp

```diff
@@ -0,0 +1,803 @@
+/*
```

Comment:
Name it something else. Dist_frien... is not a good name. Also, why can't this be just another algorithm?

Reply:
Agree. Call it `multi-phase` or something like that. And to review, it would be useful to commit the old one first and then add the new one in a different commit. If you have already done that, can you give the commit SHA?